src/pages/Product/Product.js

```diff
@@ -92,6 +93,7 @@ class Product extends Component {
                   id={product.id}
                   name={product.name}
                   collection={product.collection}
+                  sub_category={product.sub_category}
```

Comment:
```suggestion
                  subCategory={product.sub_category}
```
넘어오는 키값이 스네이크 케이스라고 하더라도 받아와서 사용할 때는 카멜케이스가 되어야 하지 않나 싶어요